test/configs/k8s/busybox-regional-rbd-deploy.yaml

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+---
+repo: https://github.com/nirs/ocm-ramen-samples.git
+path: k8s/busybox-regional-rbd-deploy/sub
+branch: test
+name: busybox-regional-rbd-deploy
+namespace: busybox-regional-rbd-deploy
+dr_policy: busybox-regional-rbd-deploy
```

Comment:
We can remove namespace and dr_policy and use name to simplify the configuration.